lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -40,15 +40,15 @@ def self.globals
       hash
     end
 
-    def fabricpath_feature
+    def self.fabricpath_feature
```

Comment:
So that it can be re-used in other classes like interface, vlan, fabricpath_topology etc., which need to check if the fabricpath feature is enabled, instead of repeating the code there. I thought you guys suggested that when I tried to use mixins.

Thx,
deepak

From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-node-utils <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, November 11, 2015 at 8:01 AM
To: cisco/cisco-network-node-utils <cisco-network-node-utils@noreply.github.com<mailto:cisco-network-node-utils@noreply.github.com>>
Cc: Cisco Employee <dcherian@cisco.com<mailto:dcherian@cisco.com>>
Subject: Re: [cisco-network-node-utils] adding fabricpath topo and fixed feature_set as a class method (#75)


In lib/cisco_node_utils/fabricpath_global.rb<https://github.com/cisco/cisco-network-node-utils/pull/75#discussion_r44548502>:

> @@ -40,15 +40,15 @@ def self.globals
>        hash
>      end
>
> -    def fabricpath_feature
> +    def self.fabricpath_feature


What is the rational behind the change from instance to class method?

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-node-utils/pull/75/files#r44548502>.